### PR TITLE
Missing .

### PR DESCRIPTION
**Pull request by tmpreston**
# Title

Missing . seperator.

### CODE REVIEW

File: xml/System.Net.Sockets/SocketType.xml [hunk 1]
```diff
@@ -66,7 +66,7 @@
...
+        <summary>Supports datagrams, which are connectionless, unreliable messages of a fixed (typically small) maximum length. Messages might be lost or duplicated and might arrive out of order. A <see cref="T:System.Net.Sockets.Socket" /> of type <see cref="F:System.Net.Sockets.SocketType.Dgram" /> requires no connection prior to sending and receiving data, and can communicate with multiple peers. <see cref="F:System.Net.Sockets.SocketType.Dgram" /> uses the Datagram Protocol (<see cref="F:System.Net.Sockets.ProtocolType.Udp" />) and the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" />.<see cref="T:System.Net.Sockets.AddressFamily" />.</summary>
```
Comment (svick) [inline xml/System.Net.Sockets/SocketType.xml]: I don't think this description was missing a `.`, I think it was missing a space:

`<see cref="F:System.Net.Sockets.SocketType.Dgram" /> uses the Datagram Protocol (<see cref="F:System.Net.Sockets.ProtocolType.Udp" />) and the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" /> <see cref="T:System.Net.Sockets.AddressFamily" />.`

Also, `Socket​Type.​Stream` has a very similar issue.

Comment (rpetrusha) [inline xml/System.Net.Sockets/SocketType.xml]: I think that this was a failed attempt to create a type/member name, @svick, so I suspect that the period is missing. At the same time, either adding a period or adding a space doesn't do much to improve this topic, which seems to me to have a certain lack of specificity. What I'd like to do, @tmpreston and @svick, is leave this PR open until I have a chance to revise the topic slightly. I'll then close this PR, open the new PR, and cc both of you to review the new PR.

Comment (svick) [inline xml/System.Net.Sockets/SocketType.xml]: @rpetrusha 

I thought that this was trying to say (in simple markdown):

`` `SocketType.Dgram` uses the Datagram Protocol (`ProtocolType.Udp`) and the `InterNetwork` `AddressFamily`.``

If it was supposed to be a type/member name, then I think the definite article would not be there:

`` `SocketType.Dgram` uses the Datagram Protocol (`ProtocolType.Udp`) and `AddressFamily.InterNetwork`.``

Also, the space is there in [an older version of this doc on MSDN](https://msdn.microsoft.com/en-us/library/system.net.sockets.sockettype(v=vs.100).aspx).

Though I'm not a native speaker and I'm arguing about what it was supposed to be when first written, not about what it should be now, so I'm fine with your suggestion.

**Comment by rpetrusha** [comment]
Actually, @svick, you're right. InterNetwork.AddressFamily is not a type/member name -- they're reversed, which I failed to notice.

**Comment by mairaw** [comment]
@rpetrusha what's the action item here?

**Comment by rpetrusha** [comment]
@tmpreston, @svick, I apologize that I've allowed this PR to languish for so long. I've finally addressed some of the issues in the documentation and opened PR #2925. I'll close this PR now.

**Pull request closed without merging**